### DISCUSSION TITLE
How can I synchronize code animations with other fragments?

### DISCUSSION THREAD

**Discussion by kescobo** [quarto-dev/quarto-cli#7601] [Q&A] [answered]
### Description

Based on [this discussion](https://github.com/hakimel/reveal.js/issues/3055) on the revealjs repo, it seems this should be possible. Here's what I tried:

````qmd
## Demo

```{.sh code-line-numbers="1|2|3" fragment-index="1" }
first
second
third
```

::: {.fragment fragment-index=1}
- Don't do this
:::

::: {.fragment fragment-index=2}
- or this
:::
````

But this leads to 2 empty animations, then lines 2 and 3 of the code, then the two bullets.

**Comment by kescobo** [comment]
Note: I can get what I need using multiple slides and auto-animate, but it would be nice to have less boilerplate to make it work

**Comment by cderv** [comment, marked as answer]
I believe there is a feature request related to that .

* https://github.com/quarto-dev/quarto-cli/issues/7378
